Code/GraphMol/Fingerprints/TopologicalTorsionGenerator.cpp

```diff
@@ -44,17 +44,29 @@ OutputType TopologicalTorsionAtomEnv<OutputType>::getBitId(
     FingerprintArguments<OutputType> *,  // arguments
     const std::vector<std::uint32_t> *,  // atomInvariants
     const std::vector<std::uint32_t> *,  // bondInvariants
-    const AdditionalOutput *,            // additionalOutput
-    const bool                           // hashResults
-) const {
+    const AdditionalOutput *additionalOutput,
+    const bool,  // hashResults
+    const std::uint64_t fpSize) const {
+  if (additionalOutput) {
+    auto bitId = fpSize ? (d_bitId % fpSize) : d_bitId;
```

Comment:
Again, for readability, maybe:
```suggestion
    auto bitId = d_bitId;
    if (fpSize) {
        bitId %= fpSize;
    }
```